src/plugins/persistence/couch/CouchObjectProvider.js

```diff
@@ -47,6 +47,7 @@ export default class CouchObjectProvider {
         let provider = this;
         let sharedWorker;
 
+        // eslint-disable-next-line no-undef
         const sharedWorkerURL = `${this.openmct.getAssetPath()}${__OPENMCT_ROOT_RELATIVE__}couchDBChangesFeed.js`;
```

Comment:
I don't think I understand this linting error. 

The definition for this in webpack is `__OPENMCT_ROOT_RELATIVE__: `'${devMode ? 'dist/' : ''`
So it should never be undefined, right?

Reply:
I gotchu @jvigliotta 
https://eslint.org/docs/user-guide/configuring/language-options#specifying-globals